frontend/src/app/AppContext.ts

```diff
@@ -1,23 +1,22 @@
 import * as React from 'react';
-import { blankDashboardCR } from '../utilities/useWatchDashboardConfig';
 import { BuildStatus, DashboardConfig } from '../types';
 
 type AppContextProps = {
   isNavOpen: boolean;
   setIsNavOpen: (isNavOpen: boolean) => void;
   onNavToggle: () => void;
   buildStatuses: BuildStatus[];
-  dashboardConfig: DashboardConfig;
+  dashboardConfig: DashboardConfig | null;
 };
 
 const defaultAppContext: AppContextProps = {
   isNavOpen: true,
   setIsNavOpen: () => undefined,
   onNavToggle: () => undefined,
   buildStatuses: [],
-  dashboardConfig: blankDashboardCR,
+  dashboardConfig: null,
 };
 
-const AppContext = React.createContext(defaultAppContext);
+export const AppContext = React.createContext(defaultAppContext);
 
-export default AppContext;
+export const useAppContext = (): AppContextProps => React.useContext(AppContext);
```

Comment:
What are you saving here by creating this?

Reply:
Talk with Juntao -- Recommended we do not do this in the future, I see no gain in this effort and it touches a lot of spots. But it is fine, we can leave it for ease of use.